components/org.wso2.carbon.identity.data.publisher.authentication.audit/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/audit/AuthenticationAuditLoggingHandler.java

```diff
@@ -123,10 +127,9 @@ protected void doPublishAuthenticationStepSuccess(AuthenticationAuditData authen
 
         AUDIT_LOG.info(String.format(
                 FrameworkConstants.AUDIT_MESSAGE,
-                authenticationData.getAuthenticatedUser(),
+                username,
```

Comment:
shouldn't it be the user id for the initiator?

Reply:
This log is created before the user gets authenticated. Therefore userId is not present in this case.